pages.fr/common/ansible.md

```diff
@@ -0,0 +1,29 @@
+# ansible
+
+> Gestionnaire de groupes d'ordinateurs à distance depuis SSH.
+> Utiliser le fichier `/etc/ansible/hosts` pour ajouter de nouveaux groupes/hôtes.
+> Plus d'information : <https://www.ansible.com>.
+
+- Lister les hôtes appartenant à un groupe :
+
+`ansible {{group}} --list-hosts`
+
+- Ping d'un groupe d'hôtes en invoquant le module "ping" :
+
+`ansible {{group}} -m ping`
+
+- Afficher des informations sur un groupe d'hôtes en invoquant le module "setup" :
+
+`ansible {{group}} -m setup`
+
+- Exécuter une commande sur un groupe d'hôtes en invoquant le module "command" avec des paramètres :
+
+`ansible {{group}} -m command -a '{{my_command}}'`
```

Comment:
Did you forget to translate this (and the next occurences), @patricedenis?

Reply:
yes sorry, I will fix that soon.
By the way, how can I add the unbreakable space by myself using Firexfox on edition?

Reply:
If you're typing the unbreakable space but it's being converted to the normal space, I wonder if it's Firefox.  Can you try Chrome?  I'm able to paste unbreakable spaces into the GitHub UI editor and they work fine using Chrome (e.g. https://github.com/tldr-pages/tldr/pull/5494/commits/8e4c89700089872e2999e9bdb54846b6d7f965d1).

Reply:
Ok, I will try this. Cheers. When I copy/paste the text into LibreOffice Writer, I can see the unbreakable white spaces, this is not the case with the Github UI I am currently using apparently.

Reply:
Sorry again, I'm a new contributor so I realises some changes but when I commited them, I got a message saying that you edited the file between my changes. I'm OK with that but the fact is I can't see where can I sync and merge your commits in my branch so I can then do the fix at the end. I may need to do another pull request?

Reply:
Are you doing it locally now?  Can I see the message?

Reply:
I'm using the Web GUI, yes

Reply:
Can you please send a screenshot?  That will help the the most.

Reply:
![image](https://user-images.githubusercontent.com/9151331/112364829-3cbfeb80-8cd7-11eb-9833-c469e995c7b5.png)
![image](https://user-images.githubusercontent.com/9151331/112364930-59f4ba00-8cd7-11eb-908e-8897851d9d03.png)

then i edit the content of the file :+1: 
![image](https://user-images.githubusercontent.com/9151331/112365039-785ab580-8cd7-11eb-9533-6e7f426b1bf5.png)
 and at the end I try to commit : 
![image](https://user-images.githubusercontent.com/9151331/112365171-a0e2af80-8cd7-11eb-8e05-44b261b84d88.png)

And I got this error message...

Reply:
But as a matter of fact, when I retried it, it didn't show up again.
So I think this is solved for now, isn't it?

Reply:
This time the commit seemed to go OK

Reply:
Yeah, I was doing some experimenting on this branch, and so I made a few commits, but then I deleted them and force-pushed, so that's why it went back.  Thank you for the details explanation!

Reply:
All right.
So I understand now it just needs the two approvals to be merged?
Anyway thank you very much for all your feedbacks.
See you next, I'm now going AFK.
Bye

Reply:
Yes, it needs two approvals.  I'll add mine. Bye :)